### ISSUE TITLE
A11y_AzureDataStudioAAE_Azure ADC(Preview)_Azure Configuration_ScreenReader: Screenreader is not announcing “selected” word for by default selected list item while navigating with help of tab key .

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15803]
https://user-images.githubusercontent.com/85863999/127464839-60272ddf-dd9a-4fbb-a10c-037611e2e6fd.mp4

### Actual:
When focus is on 'Azure account'. NVDA is not announcing “selected” word for by default selected list item while navigating with help of tab key.
Similar issue is observed with JAWS.

Similar issue is present in below scenarios:
Azure Arc Data Controller(Preview)-Controller Configuration

### Expected:
When focus is on 'Azure account'. NVDA is should announce “selected” word for by default selected list item while navigating with help of tab key.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#STP;#WCAG1.3.1;#NVDA;#Jaws;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till "Connection" and tab to see more button
4. Tab to "New Deployment" and Azure ADC(Preview) and then tab select button
5. Deployment Prerequisite page will open and tab on next "Select from existing Kubernetes cluster" will open
6. Start NVDA
7. Tab till 'Azure account' and check how NVDA is announcing the information 

### User Impact:
Screenreader user will not able to get complete information of 'Azure account'.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by Charles-Gagnon** [comment]
@sspatil123 Please link documentation indicating where this is listed as being required. I can't find any documentation saying that - and in fact I can't even get NVDA to announce "selected" through any combination of adding `aria-selected` or `selected` attributes.

It doesn't seem like announcing selected is actually adding anything useful - the user already knows it's a combobox and the currently selected item is read when tabbing onto the box which I believe should be clear enough.

**Comment by fsteffi** [comment]
this bug can be closed ad per the above comment

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (191)](https://user-images.githubusercontent.com/85863543/150975452-77c7b399-136a-4e98-a5c1-c662f0cb3b64.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
